here-naksha-app-service/src/test/java/com/here/naksha/app/service/SourceIdHandlerApiTest.java

```diff
@@ -70,6 +73,99 @@ void tc2001_testCreateFeaturesWithSourceIdInMetaWithTags() throws Exception {
                 .hasUuids();
 
     }
+    @Test
+    void tc2002_testCreateFeaturesWithoutSourceIdButWithSourceIdTag() throws Exception {
+
```

Comment:
minor styling remark: consider removing unnecessary empty lines (like the one starting / ending the method)